### ISSUE TITLE
yarn audit: critical vulnerabilities

### ISSUE THREAD

**Issue by wagner-certat** [Intevation/intelmq-fody#80]
yarn audit gives with the current lock file:
```
918 vulnerabilities found - Packages audited: 6505
Severity: 330 Low | 62 Moderate | 521 High | 5 Critical
```

**Comment by bernhardreiter** [comment]
Thanks for the issue, most of the positives will not be relevant as they are for building fody.
And even the code ending up in fody is not very exposed and probably is behind authentification
and VPN in most cases.

**Comment by bernhardreiter** [comment]
Solved with v0.7.0.
